README.md

```diff
@@ -109,7 +120,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
+BhathootFSM.jpg
```

Comment:
You diagram will work, but it contains an unnecessary state: you can merge S1 and S2:

![image](https://user-images.githubusercontent.com/10648/78616722-9ca95200-783a-11ea-9902-943bea03d655.png)

# -2